### PR TITLE
Replace crowdin-download and crowdin-upload with new crowdin-sync which points to a shared workflow

### PR DESCRIPTION
**Pull request by ben-wharton**
## Context

See my [post](https://wise.slack.com/archives/C5NKNT34H/p1706090400773849) in dev-updates.

We align this repo with current best practice so that new repositories can easily set up Crowdin.

## Checklist
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: workflow-templates/crowdin-sync.properties.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Crowdin sync",
+  "description": "For syncing sources and translations with Crowdin.",
+  "iconName": "crowdin",
+  "filePatterns": [
+      "^crowdin.ya?ml$"
```
Comment (louis-tw) [inline workflow-templates/crowdin-sync.properties.json:6]: isn't it the a who is? instead of the m?

Comment (louis-tw) [reply workflow-templates/crowdin-sync.properties.json:6]:
```suggestion
      "^crowdin.y?aml$"
```

Comment (louis-tw) [reply workflow-templates/crowdin-sync.properties.json:6]: Ah or ? applies on the letter after actually, not before 😄

Comment (ben-wharton) [reply workflow-templates/crowdin-sync.properties.json:6]: I just copied from `workflow-templates/crowdin-download.properties.json`, I actually don't know what this option does

Comment (ben-wharton) [reply workflow-templates/crowdin-sync.properties.json:6]: Anyway looks like a regex, and regex ! and ? characters modify the preceding character, right?

**Review by ltang-wise — APPROVED** [review]

**Pull request merged**